tpot/base.py

```diff
@@ -79,12 +80,15 @@ def handler(dwCtrlType, hook_sigint=_thread.interrupt_main):
 class TPOTBase(BaseEstimator):
     """Automatically creates and optimizes machine learning pipelines using GP."""
 
+    # dont save periodic pipelines more often than this
+    OUTPUT_BEST_PIPELINE_PERIOD_SECONDS = 30
```

Comment:
Should this be a TPOT parameter? Thoughts @weixuanfu2016 and @teaearlgraycold? I suppose I'm fine with a hard-coded 30 seconds, but I can see a user down the line wanting to tweak this value.